src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -838,9 +843,21 @@ public void process(Alert alert) {
 			processAlerts(
 					this.getParam(params, PARAM_BASE_URL, (String) null), 
 					this.getParam(params, PARAM_START, -1), 
-					this.getParam(params, PARAM_COUNT, -1), counter);
+					this.getParam(params, PARAM_COUNT, -1), 
+					this.getParam(params, PARAM_RISK, -1), counter);
 			
 			result = new ApiResponseElement(name, Integer.toString(counter.getCount()));
+		} else if (VIEW_ALERTS_SUMMARY.equals(name)) {
+			final ApiResponseList resultList = new ApiResponseList(name);
+			Map<String, Object> alertData = new HashMap<>();
+			int[] riskIdArray = {Alert.RISK_HIGH, Alert.RISK_MEDIUM, Alert.RISK_LOW, Alert.RISK_INFO};
+			CounterProcessor<Alert> counter = new CounterProcessor<>();
+			for (int riskId:riskIdArray) {
+				processAlerts(this.getParam(params, PARAM_BASE_URL, (String) null), -1, -1, riskId, counter);
+				alertData.put(Alert.MSG_RISK[riskId], String.valueOf(counter.getCount()));
+			}
+			resultList.addItem(new ApiResponseSet<Object>("risk", alertData));
```

Comment:
Wrapping in a list does not seem right (we have just one set). I'd suggest removing the list (this means that the JSON object needs some tweaks, there are other endpoints doing the same).